docs/examples/01-exploring-wellcome-collections-apis.md

```diff
@@ -1,6 +1,6 @@
 # 1. Exploring Wellcome Collection's APIs
 
-[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb)
```

Comment:
Are these changes here to prove that Buildkite does not get invoked on this branch?

Reply:
This the notebook update workflow doing its thing, expecting a change to notebooks rather than buildkite directory ... I _think_ this is intended behaviour to make notebook changes in a branch preview-able, but redundant here.